specs/capella/light-client/sync-protocol.md

```diff
@@ -33,6 +35,16 @@ Additional documents describes the impact of the upgrade on certain roles:
 | - | - |
 | `EXECUTION_PAYLOAD_INDEX` | `get_generalized_index(BeaconBlockBody, 'execution_payload')` (= 25) |
 
+
+## Helper types
+
+### `ExecutionBranch`
+
+```python
+class ExecutionBranch(Vector[Bytes32, floorlog2(EXECUTION_PAYLOAD_INDEX)]):  
+    pass
```

Comment:
It is exactly what we convert the "custom types" into.

What do you think about just adding custom type `ExecutionBranch: Vector[Bytes32, floorlog2(EXECUTION_PAYLOAD_INDEX)])`?